terraform/stacks/agha_infra/main.tf

```diff
@@ -75,6 +76,7 @@ resource "aws_s3_bucket" "agha_gdr_staging_2" {
     }
   )
 }
+
 resource "aws_s3_bucket_public_access_block" "agha_gdr_staging_2" {
```

Comment:
Do we still need an explicit public access block setting?
(this should be the default now, no?)

Reply:
Yes, this is the existence resource. perhaps we could remove this

Reply:
Remove it and see how it goes. 
I think we had an SCP that prevents changing this (but it's under our control), for UoM accounts it won't be, so good to get things aligned where we can.